features/search-replace.feature

```diff
@@ -62,6 +63,29 @@ Feature: Do global search/replace
     When I run `wp search-replace foo bar --quiet`
     Then STDOUT should be empty
 
+  Scenario: Verbose search/replace
+    Given a WP install
+    And I run `wp post create --post_title='Replace this text' --porcelain`
+    And save STDOUT as {POSTID}
+
+    When I run `wp search-replace 'Replace' 'Replaced' --verbose`
+    Then STDOUT should contain:
+      """
+      Updating wp_posts.post_title from 'Replace' to 'Replaced'
```

Comment:
Good question. If the intention of the verbose flag is simply to report progress, it probably only needs to explain the table and columns it's working on. Might also be nice to include whether or not it found anything to replace as it goes. Something like this:

```
$ wp search-replace 'blah' 'blurp' --verbose

Checking: wp_posts.post_content
0 rows affected
Checking: wp_posts.post_title
0 rows affected
Checking: wp_posts.post_excerpt
1 row affected
[...]
```